core/packages/analytics/pwa/components/index.js

```diff
@@ -11,12 +13,16 @@ const Analytics = ({
   gtmClientProperties,
   gtmPageViewProperties,
   gaIds,
+  gaCustomDimensions,
   comScoreIds,
-}) => (
-  <Fragment>
-    <script
-      dangerouslySetInnerHTML={{
-        __html: `
+  title,
+}) => {
+  const { title: _, ...gtmPageView } = gtmPageViewProperties;
```

Comment:
What happens if the `fetchHeadContent` fails? Do we still get the title from `entity` as a fallback?

Reply:
@DAreRodz take a look at this (I'm not sure why it says it is outdated, because it's not).